lib/Jsrt/Jsrt.cpp

```diff
@@ -2697,24 +2699,31 @@ JsErrorCode RunSerializedScriptCore(const wchar_t *script, JsSerializedScriptLoa
     });
 }
 
+#ifdef _WIN32
+template
+JsErrorCode RunSerializedScriptCore<JsSerializedScriptLoadSourceCallback, JsSerializedScriptUnloadCallback>(const wchar_t *script, JsSerializedScriptLoadSourceCallback scriptLoadCallback,
+    JsSerializedScriptUnloadCallback scriptUnloadCallback, unsigned char *buffer, JsSourceContext sourceContext,
+    const wchar_t *sourceUrl, bool parseOnly, JsValueRef *result);
```

Comment:
Hmm, thought I needed this at some point but removing it doesn't seem to make any difference- so removed it.